src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,9 +1,49 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.model.FruitTransaction;
```

Comment:
The `model` classes should be in their own package, not within the `core.basesyntax` package.